commands.js

```diff
@@ -1768,6 +1768,52 @@ exports.commands = {
 	slowchathelp: ["/slowchat [number] - Sets slowchat in the room for [number] seconds. Must be between 2-60 seconds. Requires @ * # & ~",
 		"/slowchat [off/disable] - Disables slowchat in the room. Requires @ * # & ~"],
 
+	stretching: function (target, room, user) {
+		if (!target) return this.sendReply("Stretching in this room is currently set to: " + (room.stretching ? room.stretching : false));
+		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (!this.can('declare', null, room)) return false;
```

Comment:
That's fine, but players will be able to use this then in battles.

If that's the case, should I disable this command for being used in battles?